config/install/user.role.az_content_admin.yml

```diff
@@ -13,7 +13,7 @@ permissions:
   - 'access tokens overview'
   - 'administer author entities'
   - 'administer block classes'
-  - 'administer blocks'
```

Comment:
Is this on purpose?

Reply:
Yes.  That permission used to be needed because there weren't more granular admin permissions for blocks.  Now there are (e.g. `administer block content` which is being added).